src/Symfony/Component/ExpressionLanguage/Tests/Node/ConstantNodeTest.php

```diff
@@ -40,4 +40,17 @@ public function getCompileData()
             array('array(0 => 1, "b" => "a")', new ConstantNode(array(1, 'b' => 'a'))),
         );
     }
+
+    public function getDumpData()
+    {
+        return array(
+            array('false', new ConstantNode(false)),
+            array('true', new ConstantNode(true)),
+            array('null', new ConstantNode(null)),
+            array(3, new ConstantNode(3)),
+            array(3.3, new ConstantNode(3.3)),
+            array('"foo"', new ConstantNode('foo')),
+            array('{0: 1, "b": "a", 1: true}', new ConstantNode(array(1, 'b' => 'a', true))),
```

Comment:
would be nice adding a test with a key and value that need escaping, and another with an indexed array